buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -290,12 +259,60 @@ private static void autoConfigureClusterDependencies(
         Project rootProject,
         NamedDomainObjectContainer<ElasticsearchCluster> container
     ) {
+        // Download integ test distribution from maven central
+        MavenArtifactRepository mavenCentral = project.getRepositories().mavenCentral();
+        mavenCentral.content(spec -> {
+            spec.includeGroupByRegex("org\\.elasticsearch\\.distribution\\..*");
+        });
+        project.getRepositories().add(mavenCentral);
```

Comment:
This is unnecessary. The call to `RepositoryHandler.mavenCentral()` also add the repo to the container.